### PR TITLE
Add tableau-log-viewer v1.2

### PR DESCRIPTION
**Pull request by aogail**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/tableau-log-viewer.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'tableau-log-viewer' do
+  version '1.2'
+  sha256 'd203b4d7d3d4d02d2e4051d3f08e2a506462e543d233a1d952207b830e5a3a4f'
+
+  url 'https://github.com/tableau/tableau-log-viewer/releases/download/1.2/tlv-osx-1.2.zip'
```
Comment (schwedenmut) [inline Casks/tableau-log-viewer.rb]: What about using the defined version in the url. Would look like this:
url 'https://github.com/tableau/tableau-log-viewer/releases/download/#{version}/tlv-osx-#{version}.zip'

File: Casks/tableau-log-viewer.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'tableau-log-viewer' do
```
Comment (vitorgalvao) [inline Casks/tableau-log-viewer.rb]: Needs to follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md), meaning it needs to be `tlv`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/tableau-log-viewer.rb (hunk 2 above)
Comment (aogail) [reply Casks/tableau-log-viewer.rb]: Is there a preference for tokenizing the file name instead of the full name from docs?

Comment (vitorgalvao) [reply Casks/tableau-log-viewer.rb]: It’s not a preference, it’s a requirement. The token reference goes into detail as to why.

Comment (vitorgalvao) [reply Casks/tableau-log-viewer.rb]: It’s not a tokenization, it follows from the bundle’s name. The token reference explains this in detail, as well as the reasons. Have you not read the document? You have ticked the box:

> - [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

Comment (aogail) [reply Casks/tableau-log-viewer.rb]: Yes, I read that, I just missed that there was a discrepancy between examples I read and that doc as far as the string to start with when tokenizing. I think in this type of case, where the app file name is less descriptive than the real application name, starting with the file name results in a less useful token.

In any case, I've renamed the cask to `tlv`.

Comment (vitorgalvao) [reply Casks/tableau-log-viewer.rb]: > I think in this type of case, where the app file name is less descriptive than the real application name, starting with the file name results in a less useful token.

Not doing so results in duplicates, which is a far greater problem.

**Pull request merged**